### DISCUSSION TITLE
new branch protection rule: must pass CLA check, must be up-to-date

### DISCUSSION THREAD

**Discussion by emckean** [google/docsy#722] [General]
Hi folks! Just a heads-up: we've added a branch protection rule for the 'master' branch (look for info about us switching to 'main' soon). Pull requests now must 1) pass the google/cla check and 2) be up-to-date with the branch before merging.

Please let us know (by replying in this discussion) if you run into any problems because of this change.

Thank you!

**Comment by chalin** [comment]
> Pull requests now must ... be up-to-date with the branch before merging.

👍 

> ... look for info about us switching to 'main' soon

🎉
